barretenberg/noir/bb_proof_verification/src/lib.nr

```diff
@@ -0,0 +1,43 @@
+pub global ULTRA_HONK_IDENTIFIER: u32 = 1;
+pub global ULTRA_HONK_VK_SIZE: u32 = 112;
+pub global ULTRA_HONK_PROOF_SIZE: u32 = 456;
```

Comment:
Main question I have on this PR is whether we can rely on these constants fully. Some of the programs name these in a way which implies that this can vary based on circuit size (I thought this UP was updated to have constant proof sizes?)

Reply:
In principle Honk VKs can vary in size but in practice we've fixed their size